backend/server.js

```diff
@@ -0,0 +1,10 @@
+require('dotenv').config();
+const mongoose = require('mongoose');
+
+const app = require('./app');
+
+const url = process.env.MONGODB_URI;
+
+mongoose.connect(url);
+
+app.listen(process.env.PORT);
```

Comment:
What if process.env.PORT not set?

Reply:
I've heard that it's better to get error in this case, then to have alternate port, but I'll add || 3000